src/epcpm/sunspectocsv.py

```diff
@@ -0,0 +1,532 @@
+import attr
```

Comment:
Since this is a new module shouldn't we be following the documentation standards that we had all discussed the other week? Module/class/function docstrings?

Reply:
Added docstrings and type hints to `staticmodbustoc.py` (aac3f04271f6bcb276ec9adde05fdc8d53f96900) and `sunspectocsv.py` (004cc5dfe8e5765d8f063afd11dec72e1e9f2840).

Reply:
Much appreciated!